metals/src/main/scala/scala/meta/internal/metals/MtagsResolver.scala

```diff
@@ -16,17 +16,34 @@ trait MtagsResolver {
   def resolve(scalaVersion: String): Option[MtagsBinaries]
   def isSupportedScalaVersion(version: String): Boolean =
     resolve(version).isDefined
+  def isSupportedInOlderVersion(version: String): Boolean
 }
 
 object MtagsResolver {
 
   def default(): MtagsResolver = new Default
 
+  /**
+   * Map of removed Scala versions since 0.11.10.
+   * Points to the last Metals version that supported it.
+   */
```

Comment:
Maybe add a comment to say that the mtags implementation of the current Metals version has to be backwards binary compatible with the Metals versions listed here? (it would be even greater to test that automatically)

Reply:
We have mima check for that with 0.11.10 set in previous artifacts. Maybe we should not modify it at all :thinking: